src/i18n/locales/en.json

```
@@ -66,7 +68,14 @@
 	},
 	"blog": {
 		"coming_soon": "More posts coming soon!",
-		"read_more": "Read More"
+		"read_more": "Read More",
+		"share": {
+			"default": "Browser share options",
+			"url": "Copy URL",
+			"tweet": "Twitter",
+			"facebook": "Facebook",
+			"rss": "rss"
```

this is not the original text:
```suggestion
			"rss": "RSS Feed"
```